src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
```

Comment:
no needs to wrap into template string
```suggestion
        My name is {name}
```